File: JAVA/driver.java
public class driver extends account {
    public driver(String name, String document, String email, String password) {
        super(name, document, email, password);
    }

    @Override
    public String toString() {
        return ("Name: " + name + ", Document: " + document + ", Email: " + email + ", Password: " + password);
    }
}

File: PYTHON/driver.py
from account import Account

class Driver(Account):
    uberCode = int

    def __init__(self, name, document, email, password, uberCode):
        super().__init__(name, document, email, password)
        self.uberCode = uberCode

    def toString(self):
        return super().toString() + ", Uber Code: " + str(self.uberCode)